Libraries/Utilities/deprecatedCallback.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule deprecatedCallback
+ * @flow
+ */
+
+'use strict';
+
+// @deprecated, will be removed on April 1 2016
```

Comment:
We probably won't remove it then because we may have others that we will get to over time.  Maybe we should pass the timeline in to the deprecatedCallback explicitly and add it to the warning that is output.